cocos/base/CCConsole.h

```diff
@@ -107,28 +180,63 @@ class CC_DLL Console
      * @address : 127.0.0.1
      */
     void setBindAddress(const std::string &address);
- 
+
 protected:
+    // Main Loop
     void loop();
+    
+    // Helpers
     ssize_t readline(int fd, char *buf, size_t maxlen);
     ssize_t readBytes(int fd, char* buffer, size_t maxlen, bool* more);
     bool parseCommand(int fd);
     
     void addClient();
+    
+    // create a map of command.
+    void createCommandAllocator();
+    void createCommandConfig();
+    void createCommandDebugMsg();
+    void createCommandDirector();
+    void createCommandExit();
+    void createCommandFileUtils();
+    void createCommandFps();
+    void createCommandHelp();
+    void createCommandProjection();
+    void createCommandResolution();
+    void createCommandSceneGraph();
+    void createCommandTexture();
+    void createCommandTouch();
+    void createCommandUpload();
+    void createCommandVersion();
 
     // Add commands here
-    void commandHelp(int fd, const std::string &args);
-    void commandExit(int fd, const std::string &args);
-    void commandSceneGraph(int fd, const std::string &args);
-    void commandFileUtils(int fd, const std::string &args);
-    void commandConfig(int fd, const std::string &args);
-    void commandTextures(int fd, const std::string &args);
-    void commandResolution(int fd, const std::string &args);
-    void commandProjection(int fd, const std::string &args);
-    void commandDirector(int fd, const std::string &args);
-    void commandTouch(int fd, const std::string &args);
+    void commandAllocator(int fd, const std::string& args);
+    void commandConfig(int fd, const std::string& args);
+    void commandDebugMsg(int fd, const std::string& args);
+    void commandDebugMsg_OnOff(int fd, const std::string& args);
+    void commandDirector_pause(int fd, const std::string& args);
+    void commandDirector_resume(int fd, const std::string& args);
+    void commandDirector_stop(int fd, const std::string& args);
+    void commandDirector_start(int fd, const std::string& args);
```

Comment:
Would that it contains an underscore in the function name?
I will understand. I'll fix.